pages/profile.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useEffect } from "react";
+import styles from "../styles/profile.module.css";
+
+import IconLabel from "./components/IconLabel";
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEnvelope,faSchool,faCheckCircle,faLock,faBoltLightning,faClock,faCalendar,faClockRotateLeft,faIdBadge,faRoad } from "@fortawesome/free-solid-svg-icons";
+
+import swal from "sweetalert";
+
+import moment from "moment";
+
+import Head from 'next/head';
+import 'bootstrap/dist/css/bootstrap.css';
+
+
+
+export default function UserProfile(): JSX.Element {
+  const [profile, setProfile] = React.useState<any>("");
+  useEffect(() => {
+    fetch("/api/profile")
+    .then((res: any)=>res.json())
+    .then((profile: any) => {
+      setProfile(profile);
+      // stores the jsx properties into the "profile" obj
+      profile["strandGradesJSX"] = Object.keys(profile.strandGrades).map((item:any)=>
+        <tr key={item}>
+          <td>{item}</td>
+          <td>{profile.strandGrades[item]}</td>
+        </tr>
+      );
+    })
+    .catch((err)=>{
+      // Sweetalert dialog fires when the profile fails to load
+      swal({title: "Unable to retrieve profile",text: "We were unable to retrieve the profile 😢",icon: "error"});
+    });
+  }, []);
```

Comment:
`Sweetalert` is awesome, I did not know about this before, so thanks for showing that to me! 👍

Perhaps we should use [Sweetalert2](https://sweetalert2.github.io/) since the package you use is now 4-year old?

```
$ npm install sweetalert2
```